### DISCUSSION TITLE
Layer Display - Both Layers and Height?

### DISCUSSION THREAD

**Discussion by MrDB42** [UnchartedBull/OctoDash#3428] [Q&A]
How can one get OctoDash to display both the Layer information (Layer X of Y) and the Height information (Height A of Z)? Or alternate between them?

I had it working once... but for the life of me I can't remember what I pressed to enable it (I had to start over due to a failed microSD card).

**Comment by jneilliii** [comment]
maybe it was one of the custom themes that allowed that to happen?

**Reply by MrDB42** [reply]
I never got around to installing a custom theme.

**Reply by jneilliii** [reply]
DisplayLayerProgress plugin is required to be able to get the layer number info for sure.

**Reply by MrDB42** [reply]
Yep. Have it.

I just can't remember what setting I checked in the plugin to change what is displayed on OctoDash. Anyone have any idea what part of the plugin formatting that OctoDash reads to display?

**Reply by jneilliii** [reply]
ah, in that plugin you have to enable the option Enable WebSocket-Communication

**Comment by MrDB42** [comment]
Yep,I have that WebSocket-Communication Enabled. 

But somewhere on that screen or some other setting screen in the plugin was a checkbox that I enabled/disabled (not sure which) that suddenly the OctoDash display changed (after a restart) to alternating from Laye #'s to Height #'s (now that I think about it).

When I first setup OctoDash, I was playing around with all the various Display Layer settings (I didn't like the little window in the lower left of my computer screen when I was in Octoprint and was trying to turn it off). Once I got that turned off, I noticed that the OctoDash display was different. I liked it and just ran with it. Never going back to figuring out what it was.

But I would like it back, please.
